docs/classic-ui/images.md

```diff
@@ -0,0 +1,246 @@
+---
+html_meta:
+  "description": "Image resolution and scaling in Plone Classic UI"
+  "property=og:description": "Image resolution and scaling in Plone Classic UI"
+  "property=og:title": "Image resolving and scaling"
+  "keywords": "Plone, Classic UI, classic-ui, image, resize, scale"
+---
+
+(classic-ui-images-label)=
+
+# Image handling
+
+The default content types and behaviors use the field `plone.namedfile.NamedBlobImage` for all images.
+
+For this field, image scaling and HTML tag creation is provided by the `plone.namedfile.scaling` module, specifically in the [`ImageScaling`](https://github.com/plone/plone.namedfile/blob/ecf33a2bc7a8c61888909bc383b3e08d80888e43/plone/namedfile/scaling.py#L350) view.
+
+Given a Dexterity content type named `news-item1` with a `plone.namedfile.NamedBlobImage` field named `image`, we can use the following APIs to access the image scales and manage scales.
+
+```{note}
+We will use the image object as context in the following examples.
+```
+
+(classic-ui-images-default-scales-label)=
+
+## Default scales
+
+In `/@@imaging-controlpanel` Plone allows you to configure which scales are available and what dimensions they should have. By default we have the following scales configured:
+
+* large 768:768
+* preview 400:400
+* mini 200:200
+* thumb 128:128
+* tile 64:64
+* icon 32:32
+* listing 16:16
+
+You can add or change scales as you like.
+
+
+(classic-ui-images-image-resolving-by-uri-label)=
+
+## Image resolving by URI
+
+Plone can resolve an image scale via URI in different ways.
+
+
+(classic-ui-images-by-field-and-scale-name-label)=
+
+### By field and scale name
+
+Given our `news-item1` with the field `image`, we can get the name scale `thumb` as follows:
+
+`http://localhost:8080/Plone/news-item1/@@images/image/thumb`
+
+To get the original image, you can leave out the scale:
+
+`http://localhost:8080/Plone/news-item1/@@images/image`
+
+
+(classic-ui-images-by-cacheable-scale-uid-name-label)=
+
+### By cacheable scale UID name
+
+When an image scale is created, it will be cached under the name `UID.jpeg` in the object annotations.
+It can be resolved as follows:
```

Comment:
```suggestion
When an image scale is created, it will be cached under the name `UID.EXT` (i.e. `f4c34254b44ba351af7393bfe0296664.jpeg`) in the object annotations.
Scaling keeps the uploaded formats, except for TIFF which ends up as JPEG.
It can be resolved as follows:
```